### ISSUE TITLE
Looking for a new lead maintainer

### ISSUE THREAD

**Issue by nebrelbug** [eta-dev/eta#31]
Hey everyone, this is @nebrelbug -- the lead maintainer of Eta and its companion project, [Squirrelly](https://squirrelly.js.org).

On October 28, 2020, I will leave on a 2-year service opportunity. During that time, I will be unable to work on any of my open-source projects.

Since my date of departure is drawing closer, I've decided it is time to begin **looking for a new lead maintainer**.

## Maintainer Requirements

- Proficiency in TypeScript
- Open-source experience. Preferably the maintainer/developer of an open-source JS/TS project with 50+ stars
- Willing to test changes & not break anything important :wink:
- Willing to follow the Project Goals (see below)

**Maintainer Jobs**

- Fix bugs
- Answer questions

A maintainer could also update documentation, write tutorials, write plugins, write integrations (ex. for Fastify, Eleventy, Koa) etc. In case this sounds overwhelming, remember that Eta is only 2.5KB minzipped :joy:

Ideally a maintainer would be willing to maintain Squirrelly as well (they share much of the same codebase) but that's definitely not a requirement.

## Project Goals

- Follow SemVer versioning guidelines
  - This means Eta's public API should remain backwards-compatible (at least within v1.x.x)
- Remain lightweight
  - Browser build should never exceed 3KB minzipped
- Remain stable
- Keep parsing engine
  - Eta's parser has been optimized quite a bit for performance and reliability, and any significant changes run too high a risk of impacting either

## Fallback Plan

There's always the possibility I will be unable to find a new lead maintainer before my departure. This is one of the reasons I've accelerated development so much. It's also the reason Eta and Squirrelly are so extensively tested.

I feel confident that Eta is stable enough it could continue to function, if needs be, without maintenance for several years. It has a stable API, is well tested, and has decent documentation. Many features can be added as 3rd-party plugins rather than core parts of the library. Moreover, Eta's small size reduces the likelihood that important bugs exist inside the code.

That scenario, though, would be far from ideal, so I've made several backup plans:

- Give repository + npm access to a friend I trust to act in my behalf. This friend would not modify Eta's code, but could give access if, while I was gone, someone asked to become a maintainer
- Add the project to https://www.codeshelter.co/

## TL;DR

I'm going to be away for 2 years and unable to maintain Eta.

In case I am unable to find a maintainer before I leave, I will give repository access to a trusted friend and add the repository to https://www.codeshelter.co/ (allowing anyone to apply for maintainership).

If you want to become one of Eta's maintainers and have experience with open-source and TypeScript, message me on [Gitter](https://gitter.im/nebrelbug), email me at nebrelbug [at] gmail [dot] com, or contact me some other way!

**Comment by shadowtime2000** [comment]
I am assuming that since @nebrelbug has already left to go on his 2 year service opportunity and I am the only maintainer who can work on Eta right now, this issue should be closed.
